packages/discovery-provider/src/tasks/entity_manager/entities/playlist.py

```diff
@@ -701,7 +744,6 @@ def process_playlist_data_event(
                 )
             else:
                 setattr(playlist_record, key, playlist_metadata[key])
-    playlist_record.last_added_to = None
```

Comment:
i think this is correct - if there are no tracks in the playlist update, we don't want to set `last_added_to` to null.